docs/workforce-management-framework/topics/con-extending-the-user-interface.adoc

```diff
@@ -0,0 +1,57 @@
+[id='{context}-con-extending-the-user-interface']
+= Extending the User Interface
+
+{Feature} framework provides a set of extension points which allows a developer to modify an Angular User Interface (UI).
+All modifications are done by supplying additional modules which provide a set of the Angular.js directives and are used to extend the Mobile and Portal UI.
+The {Feature} Application is shipped with the example `angularjs-extensions` example that extends the Demo UI:
+
+ * https://github.com/feedhenry-raincatcher/raincatcher-angularjs/tree/master/packages/angularjs-extensions
+
+This Module contains the example directives that are used for demo purposes and it can be used as a template to extend the existing {Feature} UI.
+
+== Extending Workorders
+
+By default, {Feature} provides a minimal set of defined fields for a _workorder_.
+Depending on the needs of the business, a developer can add additional data and UI elements, for example, gps location, an address or a description UI element.
+
+=== Building Workorders
+To extend a _workorder_, implement two Angular Directives:
+
+.Angular Directives for Extending a _workorder_
+|===
+|Angular Directive |Description
+
+|`workorder-data`
+|A Data directive used to implement static fields that display data.
+
+|`workorder-data-edit`
+|A Data-edit directive used to implement fields that allow modifications to be made to a _workorder's_ data.
+
+|===
+
+NOTE: Directives are to operate (with respect to existing fields) outside of a _workorder_ to avoid naming conflicts, for example, `workorder.data.yourcustomfield`
```

Comment:
@wtrocki - can you take a look at this and make sure I haven't lost the technical context as a result of my change?